packages/rum/package.json

```diff
@@ -15,7 +15,7 @@
     "karma": "karma start",
     "karma-coverage": "karma start --coverage",
     "serve": "npm run script serveE2e ./ 8000",
-    "runUnitTests": "npm run script runUnitTests",
+    "runUnitTests": "karma start --single-run",
```

Comment:
Previously we did this with `karma start` but IMO using the script that we wrote gives us more control!

Reply:
This still uses the config from `karma.config.js` so I believe we control everything from there. The script we had effectively did `singleRun: true`. We were not using any custom functionality so I felt its better to remove it altogether.

Reply:
Let's keep using our own scripts, In most cases making a small changes is more easier for us than with using original script!